ap-apps-intros (#210)
gear-tech/wiki: docs/api/getting-started.mdx

## Getting started

> Start the API connection by running a local RPC node:

Reviewer: I guess we connect to the already running node, aren't we?
```suggestion
Start the API connection to the local running RPC node:
```
